search_gov_crawler/search_gov_spiders/spiders/domain_spider_js.py

```diff
@@ -0,0 +1,71 @@
+from typing import Optional
+
+from scrapy.spiders import CrawlSpider, Rule
+from scrapy.http import Response, Request
+
+import search_gov_crawler.search_gov_spiders.helpers.domain_spider as helpers
```

Comment:
The absolute imports did not work for me, but using the relative imports (i.e from ..helpers import domain_spider as helpers) did